source/en/api_classifier.rst

```diff
@@ -226,11 +226,11 @@ Methods
       Estimates labels from given ``data``.
       This API is designed to accept bulk classification with list of :mpidl:type:`datum`.
 
-   .. mpidl:method:: list<string> get_labels()
+   .. mpidl:method:: map<string, int> get_labels()
 
-      :return:     list of all labels in the jubatus
+      :return:     Pairs of label and the number of trained data
 
-      Returns all label list.
+      Returns the number of trained data for each label.
```

Comment:
The expression `number of trained data` is ambiguous. Need to mention the unlearner behavior?